specification/security/resource-manager/common/v1/types.json

```diff
@@ -65,7 +65,7 @@
           "$ref": "#/definitions/Resource"
         },
         {
-          "$ref": "#/definitions/Location"
+          "$ref": "#/definitions/LocationReadAndWrite"
```

Comment:
TrackedResourceLocation would better describe the motivation to add this type IMO